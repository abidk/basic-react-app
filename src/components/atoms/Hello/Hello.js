import React, { memo } from 'react';
import PropTypes from 'prop-types';
import './Hello.css';

const Hello = (props) => <p className="HelloText">Hello {props.name}</p>

Hello.propTypes = {
    name: PropTypes.string.isRequired
}

export default memo(Hello);