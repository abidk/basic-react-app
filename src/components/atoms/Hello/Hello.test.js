import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Hello from './Hello';

describe(__filename, () => {
    it('Test Hello render', () => {
        const output = shallow(<Hello name="Abid" />);

        expect(shallowToJson(output)).toMatchSnapshot();
    });
});