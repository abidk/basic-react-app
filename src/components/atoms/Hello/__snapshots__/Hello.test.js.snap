// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/Users/abidkhalil/Documents/basic-react-app/src/components/atoms/Hello/Hello.test.js Test Hello render 1`] = `
<p
  className="HelloText"
>
  Hello 
  Abid
</p>
`;
